src/main/kotlin/app/widgets/recordButton/RecordingAnimation.kt

```diff
@@ -0,0 +1,60 @@
+package app.widgets.recordButton;
+
+import app.UIColorsObject.Colors
+import javafx.animation.Timeline
+import javafx.geometry.Pos
+import javafx.scene.layout.HBox
+import javafx.scene.paint.Color
+import javafx.scene.shape.Arc
+import javafx.scene.shape.ArcType
+import javafx.scene.shape.Circle
+import tornadofx.*
+import java.time.Duration
+
+class RecordingAnimation : HBox() {
+    var animation: Timeline? = null
+    var arc: Arc? = null
+    var circle: Circle? = null
```

Comment:
why are these nullable?

Reply:
they shouldn't be. I was trying to do some refactoring on that and should have changed it back.